package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-oauth2",
-  "version": "2.3.1",
+  "version": "2.3.2",
```

Comment:
Please leave at 2.3.1. I will update the version when doing  the next release (which would be 2.4.0)